src/script/run-make.sh

```diff
@@ -54,6 +54,16 @@ function prepare() {
         which_pkg="debianutils"
     fi
 
+    if in_jenkins; then
+        if ! type clang-16 > /dev/null 2>&1 ; then
+            ci_debug "Getting clang-16"
+            wget https://apt.llvm.org/llvm.sh
```

Comment:
So, it looks like llvm.sh only supports Debian/Ubuntu, but it seems like the rest of run-make-check.sh etc. still supports RPM systems.  Should this section of code be made dependent on debian/ubuntu?

Reply:
updated